docs/root/api/starting_envoy.rst

```diff
@@ -135,6 +133,36 @@ Note that the library automatically flushes stats on application backgrounding/t
   // Swift
   builder.addStatsFlushSeconds(5)
 
+~~~~~~~~~~~~~~~~~~~~~~~~
+``addAppVersion``
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+Specify the version of the app using Envoy Mobile. This information is sent as identifying
+information when flushing metrics.
+
+**Example**::
+
+  // Kotlin
+  builder.addAppVersion("version")
+
+  // Swift
+  builder.addAppVersion("version")
+
+~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
These seem excessively long 🤔 didn't know this was valid

Reply:
trimmed them. But yeah, restructuredtext actually does not require them to be the width of the text. Interestingly, they also don't require specific characters to denote specific heading levels...